mss/__meta__.py

```diff
@@ -2,6 +2,6 @@
 Meta values for package.
 """
 
-__version__ = '1.8.0'
+__version__ = '1.8.1'
```

Comment:
C'est pas incohérent avec 1.9.1 ?
```suggestion
__version__ = '1.9.1'
```